src/styles/molecules/_molecules.button-group.scss

```diff
@@ -69,15 +69,15 @@
 .m-button-group--vertical {
   flex-direction: column;
 
-  > .a-button {
+  > .a-button:not(.a-button--outlined) {
```

Comment:
Kunnen we dit (eventueel met iets meer code) zonder `:not` pseudo selector doen?